internal/scanner/scanner.go

```diff
@@ -231,7 +229,7 @@ func defaultScanner() Scanner {
 	// Using a function rather than a global is intentional; this function is
 	// inlined as pure code (zeroing + moves), whereas a global requires write
 	// barriers since the memory is mutable.
-	return Scanner{languageVersion: core.ScriptTargetLatest, skipTrivia: true}
+	return Scanner{skipTrivia: true}
```

Comment:
If I make skipTrivia a method, then flip the default, then we can also reset scanners with zeroing, which would be amazing.